api/test/test_helper/fake_android_plugin.gd

```diff
@@ -38,8 +43,25 @@ func remove_asset_location(_pack_name : String, _asset_path : String):
 func clear_asset_location_store():
 	_asset_location_store.clear()
 
+func add_asset_pack_location(_pack_name : String, _asset_pack_location_dict : Dictionary):
+	_asset_pack_location_store[_pack_name] = _asset_pack_location_dict
+
+func remove_asset_pack_location(_pack_name : String):
+	_asset_pack_location_store.erase(_pack_name)
+
+func clear_asset_pack_location_store():
+	_asset_pack_location_store.clear()
+
+# -----------------------------------------------------------------------------
+# Mock Functions
+# -----------------------------------------------------------------------------
 func getAssetLocation(_pack_name : String, _asset_path : String):
 	var dict_key = [_pack_name , _asset_path]
 	if dict_key in _asset_location_store:
 		return _asset_location_store[dict_key]
 	return null
+
+func getPackLocation(_pack_name : String):
+	if _pack_name in _asset_pack_location_store:
```

Comment:
Do you need this if check?

Reply:
This check is required since for GDScript accessing a non-existent key in Dictionary will trigger an 
```
Invalid get index $key (on base: Dictionary)
```
runtime error.